extensions/wikia/UserRenameTool/SpecialRenameuser.php

```diff
@@ -42,10 +42,28 @@
 
 // classes
 $wgAutoloadClasses['SpecialRenameuser'] = dirname( __FILE__ ) . '/SpecialRenameuser_body.php';
-$wgAutoloadClasses['RenameUserHelper'] = dirname( __FILE__ ) . '/RenameUserHelper.class.php';
 $wgAutoloadClasses['RenameUserProcess'] = dirname( __FILE__ ) . '/RenameUserProcess.class.php';
 $wgAutoloadClasses['RenameUserLogFormatter'] = dirname( __FILE__ ) . '/RenameUserLogFormatter.class.php';
-$wgAutoloadClasses['UserRenameTask'] = dirname( __FILE__ ) . '/UserRenameTask.class.php';
+
+// Resource Loader modules
+$userRenameResourceTemplate = array(
+	'localBasePath' => __DIR__ . '/modules',
+	'remoteExtPath' => 'wikia/UserRenameTool/modules'
+);
+$wgResourceModules['ext.renameuser.modal'] = $userRenameResourceTemplate + array(
+	'scripts' => 'ext.renameuser.modal.js',
+	'dependencies' => array(
+		'mediawiki.util'
+	),
+	'messages' => array(
+		'renameuser',
+		'userrenametool-new',
+		'userrenametool-confirm',
+		'userrenametool-confirm-intro',
+		'userrenametool-confirm-yes',
+		'userrenametool-confirm-no'
+	)
+);
 
 // constants
 // define('ENV_DEVBOX', true);//TODO: used for some debug switches, comment out as soon as the code hits production!
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)

[project-id]: wikia:app